z/zfs.yml

```diff
@@ -12,7 +12,6 @@ zfs:
 #  - /usr/bin/ros:/usr/bin/ros
   volumes_from:
   - all-volumes
-  restart: always
```

Comment:
Yes, I got into a crash loop that turned my laptop into a personal heater.  Compile for 10 minutes, fail, restart, compile for ten minutes...